src/lib/pages/account-details/index.tsx

```diff
@@ -211,11 +233,11 @@ const AccountDetailsBody = ({
             Delegations
           </CustomTab>
           <CustomTab
-            count={nftsCount}
+            count={totalNfts}
             isDisabled={nftsCount === 0}
```

Comment:
Do we need to change to totalNfts?

Reply:
Yes kub, since it use in multiple place and support both from full version and sequencer version

```js
const totalNfts = nftsCount ?? accountNfts?.total;
```